lumicks/pylake/scalebar.py

```diff
@@ -0,0 +1,153 @@
+from dataclasses import dataclass, field
+from matplotlib.offsetbox import AnchoredOffsetbox, AuxTransformBox, VPacker, HPacker, TextArea
+
+
+def _create_scale_legend(
+    transform,
+    size_x,
+    size_y,
+    label_x,
+    label_y,
+    loc,
+    color,
+    separation,
+    barwidth,
+    fontsize,
+    **kwargs,
+):
+    """Draws a scale legend for the current axis.
+
+    Parameters
+    ----------
+    transform : :class:`matplotlib.transforms.CompositeGenericTransform`
+        The coordinate frame (typically axes.transData).
+    size_x, size_y : float
+        Length of the scale bars in data units. `0` or `None` omits the scale.
+    label_x, label_y : str or None
+        Labels for the scale bars. None results in no label on the scale.
+    loc : position in the containing axis
```

Comment:
```suggestion
    loc : str
```

Reply:
maybe `position in the containing axis` can go below

Reply:
Whoops, yes, definitely. Good catch!